documentation/asciidoc/topics/proc_backing_up_cluster.adoc

```diff
@@ -0,0 +1,53 @@
+[id='backing_up_clusters-{context}']
+= Backing Up {brandname} Clusters
+Create backup files that stores the state of {brandname} clusters to persistent volume claims.
+
+.Prerequisites
+
+* Create a {brandname} cluster of {datagridservice} nodes.
+* Create at least one cache and add some entries.
+
+[IMPORTANT]
+====
+The backup operation preserves the state of {brandname} clusters.
+To prevent data loss, you should ensure there are no active client connections to the cluster you want to back up.
```

Comment:
It might be better to state something like "{brandname} backups do not provide [snapshot isolation](https://en.wikipedia.org/wiki/Snapshot_isolation), entries are stored in the archive as they are read meaning that any write operations peformed between a backup starting and finishing, may not be included in the archive. Therefore, if a backup of the exact container content is required, it's necessary to ensure ensure no active client connections exist when the backup is initiated".

Having active connections during the backup won't result in data loss from the running cluster, it just means that the backup may include an original entry value of `k=1`, whereas the value in the cluster could be updated to `k=2` during the backup process and this will only be reflected in the live cluster as the entry has already been stored in the backup archive.